### PR TITLE
Added 3 breakpoints

### LINKED ISSUE #10: Responsive design
Do all the responsive adaptations that should have been done from the start. Good luck with that, future 0k3ri! 
- Add at least three breakpoints. 
- Add print option

### LINKED ISSUE #21: Responsive design: relative units for zoomed views
All the font sizes and media queries widths are in px, check the whole code for other used absolute units.

### LINKED ISSUE #22: Contact & About Me index is broken
The links are broken, it's probably a path issue.

### LINKED ISSUE #24: Add indentation to paragraphs

### PR DESCRIPTION
**Pull request by espimadrazo**
I think the general design might be somewhat okay though I read it's recommended to use min-width on media queries instead of max-width, but I never know if I can trust the source of the comment whenever I search online for best practices. 

I'm having trouble finding out why does the hamburger icon displays well on the home page and not on every other sub page, because it's exactly the same header on each file. I have been reading the whole code for a while and still can't find the reason, I suspect it might be related to the selectors but I don't know how. 

I also have a small question regarding the order of the imported style sheets. Should the base sheet be first rather than the responsive one or is it irrelevant? I figure since the code is read in order I should probably change it because the base sheet is more important and should load first.

### CODE REVIEW

**Comment by espimadrazo** [comment]
I solved the hamburger issue by tagging  _header img.hamburger-menu_ instead of _header.hamburger-menu_ or _header .hamburger-menu_. I still think I'm probably doing something wrong because I still don't understand the reason why 

```
header .hamburger-menu {
  display: block;
}
```
worked but ONLY on home page. I'll make the commit later.

**Comment by espimadrazo** [comment]
On the latest commit you'll find a new bug on the print media query: the Hamburger Icon displays on the print sheet and the Index Title doesn't, when it should be the other way around (on the style.css sheet the first one is hidden and the second one is visible). 
I will optimize this media query after the deployment, adding paper size adjustments and others.

As I said before, I also fixed the display of the Hamburger Icon on the subpages but a general feedback on the latest changes would be much appreciated (:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This is my first personal project on HTML, CSS and JavaScript. It's a minimalist page with information about cognitive baias and psychological paradoxes.
```
Comment (mauriciopasquier) [inline README.md]: baia baia

File: about/index.html [hunk 2]
```diff
@@ -5,24 +5,31 @@
...
           >
         </a>
       </div>
 
-      <!-- TODO, Make every page consistent with this change.-->
       <menu class="index">
-        <span>Uncanny Index ↓</span>
+        <img class="hamburger-menu"
+          src="../assets/hamburger-menu.svg"
+          width="40" height="1000"
```
Comment (mauriciopasquier) [inline about/index.html]: This is too much height.

File: base-rate-fallacy/index.html [hunk 3]
```diff
@@ -5,23 +5,44 @@
...
           >
         </a>
       </div>
-    </header>
 
-    <div class="line"></div>
+      <menu class="index">
+        <img class="hamburger-menu"
+          src="../assets/hamburger-menu.svg"
+          width="40" height="1000"
```
Comment (mauriciopasquier) [inline base-rate-fallacy/index.html]: Height

File: base-rate-fallacy/index.html [hunk 4]
```diff
@@ -5,23 +5,44 @@
...
+          alt="Hamburger menu"
+        >
+        <div class="index-title">
+          <span>Uncanny Index ↓</span>
+        </div>
+        <ul class="dropdown-content">
+          <li><a href="self-fulfilling-prophesy/index.html">Self Fulfilling Prophecy</a></li>
```
Comment (mauriciopasquier) [inline base-rate-fallacy/index.html]: I think all these href should point to the base folder, right? With `../`

File: contact/index.html [hunk 5]
```diff
@@ -5,22 +5,31 @@
...
+            alt="Drawing of an owl"
           >
         </a>
       </div>
+
       <menu class="index">
-        <span> Uncanny Index ↓</span>
+        <img class="hamburger-menu"
+          src="../assets/hamburger-menu.svg"
+          width="40" height="1000"
```
Comment (mauriciopasquier) [inline contact/index.html]: Height

**Review by mauriciopasquier — CHANGES_REQUESTED** [review]
Very nice responsive menu! Check out these comments and it'll be ready to merge.

**Comment by mauriciopasquier** [comment]
> On the latest commit you'll find a new bug on the print media query: the Hamburger Icon displays on the print sheet and the Index Title doesn't, when it should be the other way around (on the style.css sheet the first one is hidden and the second one is visible).
> I will optimize this media query after the deployment, adding paper size adjustments and others.

I forgot about this issue. You have to restore back the `display: flex` on `.index-title` and hide the `.hamburguer-menu` on this media query.

**Review by mauriciopasquier — APPROVED** [review]

**Pull request merged**